### ISSUE TITLE
Setup help

### ISSUE THREAD

**Issue by Hussain-maker** [David74000/SP2019-modern-search-webpart-with-taxonomy-refiner#1]
Hi David,

Thanks for posting this solution up, I was just wondering if you have any pointers on how to use the Taxonomy refinement filter?

I'm guessing you use with the owsmetadatatagsallinfo property but I'm not successful yet. Could you help with an example or anything?

**Comment by David74000** [comment]
Hi @Hussain-maker ,
no I am not using the owsmetadatatagsallinfo. I am using a managed property that is plugged to a taxonomy column, and that is set a refinable in the search schema. The managed property is linked to a crawled property that has the following prefix : **ows_taxId_**MYCOLUMNNAME. The important thing is that your managed property contains the ID of the term. You can verify this point by using the default renfinement template first, before using the taxonomy one.
You should have something like that :

![image](https://user-images.githubusercontent.com/64143808/117413494-567b6200-af16-11eb-90bc-b2c3299c72f5.png)

Let me know if it helps
David

**Comment by JoelHJ** [comment]
I'm also curious about this. I have done what you describe and i get the data but it looks the same as with the default template. It won't drop the numberstring. I read in issue 610 that one needs to look at localization or something?

**Comment by David74000** [comment]
Hi @JoelHJ 
If you select the default template do you have all terms in the following format : GP0|#XXX,  GPP|#XXX or GTSet |#XXX where XXX is the id of the term in termstore ? The taxonomy refiner template read the format as described to get the term id and whether or not it's a parent term or a term set.
If so, when you select **Taxonomy** in the dropdown for template refiner menu, then it should display all term label in a treeview.
David

**Comment by JoelHJ** [comment]
@David74000 yes i do have it like that. However when i change the template it goes blank instead. 
L0|#045f7ee9d-fca2-48d1-bdec-875d738a3283|Collaboration (6)

It seems the template doesn't read it correctly... as when i change back to default one it reads it as above.

**Comment by JoelHJ** [comment]
@David74000 actually got it to work. Just had to change the temnplate and actually publish the page and reload it. IT works. :-)

**Comment by David74000** [comment]
Cool ! nice job !
